src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ClassifiedSpan/ClassifiedSpanVerifier.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
What's the change here? Does this remove the "BOM character"? I think this should be reverted.

Reply:
Outside of files that are part of the project templates. I don't particularly mind these sort of changes. Encoding of files isn't consistent across the repo and some of these changes often update the file encoding to match what's in .editorconfig